### PR TITLE
Feat(refs:T322012): vue3 migration - usage of template tag/element

### PR DESCRIPTION
**Pull request by elmasdemos**
**Ticket:** https://yaits.demos-deutschland.de/T32012#761740

### Description: 

- <template> tags with no special directives are now treated as plain elements and will result in a native <template>
 element instead of rendering its inner content
- remove <template> if no directives and not necessary any more
- in one commit there was a problem, that the template looks unnecessary because the slot name spelling, which shall reference to the slot of the template, was deprecated
- instead of `slot="name"` use `v-slot:name` or the more shortened way `#name` 

- ***this PR will be merged into the integration branch `f_vue3_compat_mode_setup`***

### Linked PRs

- https://github.com/demos-europe/demosplan-core/pull/1240

### PR Checklist

- [x] Link all relevant tickets

### CODE REVIEW

File: client/js/components/statement/segments/DpRecommendationModal.vue [hunk 1]
```diff
@@ -7,62 +7,58 @@
...
-  <dp-modal
-    ref="recommendationModal"
-    content-classes="u-2-of-3">
-    <template>
-      <h3>{{ Translator.trans('segment.recommendation.insert.similar') }}</h3>
-      <div class="layout u-mb">
-        <div class="layout__item u-1-of-3">
-          <span class="display--block weight--bold">
-            {{ Translator.trans('segment.tags') }}
+<dp-modal
```
Comment (muellerdemos) [inline client/js/components/statement/segments/DpRecommendationModal.vue:10]: thanks for clarifying we do not need a top-level `<template>` if the top-level component is a custom one.

**Review by muellerdemos — CHANGES_REQUESTED** [review]
thanks for the effort to migrate to vue3 so far. Could only spot 1 minor nitpick - rest lgtm

File: client/js/components/statement/statement/DpVersionHistory.vue [hunk 2]
```diff
@@ -15,55 +15,53 @@
...
+      {{ versionHistoryHeading }}
+    </h2>
+
+    <dp-loading
+      v-if="isLoading"
+      data-cy="loadingSpinner"
+      class="u-mt-1_5 u-ml" />
+
+    <div
+      v-else-if="false === isLoading"
```
[the remark this answers is not in the document]
Comment (elmasdemos) [reply client/js/components/statement/statement/DpVersionHistory.vue]: thanks for the advice. done https://github.com/demos-europe/demosplan-core/pull/1275/commits/2af8b6509373690c131132613f1cf4994f56c134

**Review by muellerdemos — APPROVED** [review]
lgtm

**Pull request merged**